src/selectors.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  SelectionEntry,
+  SelectionStateInfo,
+  UIDisplayData,
+} from '@aics/simularium-viewer';
+import {
+  ActiveState,
+  VisibilitySelectionMap,
+  ViewerVisibilityStates,
+} from './constants';
+
+const { Inactive } = ActiveState;
+
+export const getPayloadForHideAll = (
```

Comment:
I think this is getting the next state of the checkbox based on the previous state of the checkbox? I would name it something that reflects that. And I also don't think you need to be storing this. The Dom already does, so you just need to be updating the state of agents based on this change